data/governments.txt

```diff
@@ -511,6 +531,12 @@ government "Hai Merchant (Human)"
 		"Hai (Friendly Unfettered)" .1
 		"Hai (Unfettered Civilians)" .1
 		"Merchant" .01
+	"penalty for"
+		assist -.15
+	"custom penalties for"
+		"Hai (Unfettered)"
+			capture 0
+			destroy 0
```

Comment:
Why do the humans also not care about these? They already have a more negative attitude towards the Unfettered than everyone else here.

Reply:
~~Because the gap between the factions is always an issue?~~
We could remove it ig